tests/json-fixtures/test_transactions.py

```diff
@@ -40,43 +45,92 @@
 BASE_FIXTURE_PATH = os.path.join(ROOT_PROJECT_DIR, 'fixtures', 'TransactionTests')
 
 
+@to_tuple
+def expand_fixtures_forks(all_fixtures):
+    """
+    The transaction fixtures have different definitions for each fork and must be
+    expanded one step further to have one fixture for each defined fork within
+    the fixture.
+    """
+    # TODO: This was copied from state tests. Make more DRY.
+    for fixture_path, fixture_key in all_fixtures:
+        fixture = load_fixture(fixture_path, fixture_key)
+        for fixture_fork, fork_states in fixture.items():
+            if (fixture_fork in ForkName.__members__):
```

Comment:
That's a good point. The reason I did that is because the fork names are on the same level as `rlp` and `info` but I agree to your point so it would actually be safer to just skip `_info` and `rlp` but let everything else go through so that we won't miss a fork test.